yarn-project/noir-protocol-circuits-types/src/scripts/generate_client_artifacts_helper.ts

```diff
@@ -54,12 +54,12 @@ function generateCircuitArtifactImportFunction() {
       // In the meantime, this lazy import is INCOMPATIBLE WITH NODEJS
       return `case '${artifactName}': {
         const { default: compiledCircuit } = await import(\"../artifacts/${artifactName}.json\");
-        return compiledCircuit as NoirCompiledCircuit;
+        return { ...(compiledCircuit as NoirCompiledCircuit), name: '${artifactName}' };
```

Comment:
This code generates `client_artifacts_helper.ts` which looks like this after the changes:
<img width="1349" alt="image" src="https://github.com/user-attachments/assets/b2be7d67-e2b4-4839-be81-e6ba1e051b8e" />